sites/fast-website/src/app/css/style.css

```diff
@@ -122,6 +122,10 @@ site-navigation ::part(content) {
   align-items: flex-end;
 }
 
+.logo::part(content)::before {
+  background: transparent;
+}
+
```

Comment:
Another way you could manage this would be to pass the logo to the start or end slot. This would require no CSS changes as that slot doesn't get underlined. Hovering the GH link will illustrate.

Reply:
I could put it in the start slot. but it looks like the log and FAST text are both SVGs. I could do this,
`<span slot="start">
              <%= require("svg/icon-brand.svg") %>
              <%= require("svg/icon-brand-fast.svg") %>
            </span>`
but I will still need to add another class to fix the alignment.

Reply:
I'm fine with either implementation. The current one seems to be less intrusive as it currently is, so let's roll with it.